newdle/client/src/client.js

```diff
@@ -43,6 +43,13 @@ class Client {
     return this._request(flask`api.get_newdle`({code}), {anonymous: !fullDetails});
   }
 
+  updateNewdle(code, newdle) {
```

Comment:
What do you think about making this a more specific `setFinalDT(code, dt)` operation? The underlying call would not change but I think as far as the Client is concerned this is a separate operation from "just" updating a newdle (especially since that's something we don't support)

Reply:
I thought that we were going to support updating a newdle in future, so this operation could have more uses than just setting the final date.